NO_GOLD_001 (#358)
NINAnor/ecRxiv: R/NO_GOLD_001.qmd

::: {.callout-tip collapse="true"}
## Metadata
```{r tbl-meta2}
#| tbl-cap: 'Indicator metadata'
#| echo: false
#| warning: false
meta |>
  dplyr::filter(Variable %in% c(
    "Indicator ID",
    "Indicator Name",
    "Continent",
    "Country",
    "Ecosystem Condition Typology Class", 
    "Realm",
    "Biome",
    "Ecosystem",
    "Year added", 
    "Last update",
    "Version",
    "Version comment",
    "Normalised",
    "Spatial aggregation pathway")) |>   
  kable()

```

:::

::: {.callout-tip collapse="true"}

## Log

<!--# Update this logg with short messages for each update -->
- May 27 2026 - submission
:::

<hr />

<!--# Document you work below.  -->

## 1. Summary

<!--# 
With a maximum of 300 words, describe the indicator in general terms as you would to a non-expert. Think of this as a kind of common language summary. Follow the four numbers sub-headers below. under point 2, it is possible under  include a bullet point list of the specific steps in the workflow. 
 -->

1. **Background and rationale**:
<!-- What does the metric represent? Why is this relevant for describing ecosystem condition in this ecosystem?
What are the main anthropogenic impact factors? -->

> Golden eagle is common top predator in Norwegian high-altidude forests and mountains, although it also utilises a range of other habitats, from the coast to the high alpine areas. 

Reviewer: ```suggestion
Golden eagle is a common top predator in Norwegian high-altidude forests and mountains, although it also utilises a range of other habitats, from the coast to the high alpine areas. 
```

It feeds mainly on mid-range mammals and game birds.

> The golden eagle is protected from hunting in Norway, but may be negatively affected by other anthripogenic stressors, such as land use change, habitat degradation and climate change. 

Reviewer: ```suggestion
The golden eagle is protected from hunting in Norway, but may be negatively affected by other anthropogenic stressors, such as land use change, habitat degradation and climate change. 
```

2. **Methods**:
<!-- What kind of data is used? How is the data customized (modified, estimated, integrated) to fit its purpose as an indicator? -->

> The size of the golden eagle population is estimated from expert opinion, based on information available forom different sources, including the national golden eagle monotoring program. There is, however, considerable uncertatinties associated with both the reference condition/values and the current estimates.  

Reviewer: ```suggestion
The size of the golden eagle population is estimated from expert opinion, based on information available from different sources, including the national golden eagle monotoring program. There is, however, considerable uncertatinties associated with both the reference condition/values and the current estimates.  
```

3. **Interpretation**:
<!-- Add a very short description of how the indicator values should be interpreted. Here's an example for a bumblebee indicator: An indicator value of 1 signifies an intact bumblebee community. Values decreasing toward zero reflect negative ecological change, specifically the decline of species expected to be present. This metric does not account for potentially positive changes, such as increases in species' abundance or the arrival of new species. Tip: Add text on the line directly following the subheader, and don't introduce blank lines. This way the whole paragraph renders as a block quote. -->
An indicator value of 1 means that golden eagle is as numerous, or more, than what we expect under the reference condition.
Values decreasing toward zero indicates that the average population size is decreasing.

4. **Results and development needs**:
<!-- Briefly, what does the indicator say about the ecosystem condition. What is the current status of  the metric (can it be used or is it still in development), and what immediate tasks remain to improve the indicator or to make the indicator operational -->

> The indicator value for Norway as a whole in 2024 is 0.89 [95% CI = 0.80-0.97]. This is quite high, and shows that golden eagle is quite common in Norway today. The indicator value is similar in the south as in the north of the country. Development needs include better population estimates though intensified monitoring, and a revision of the X~100~ values based on updated knowledge.     

Reviewer: ```suggestion
The indicator value for Norway as a whole in 2024 was 0.89 [95% CI = 0.80-0.97]. This is quite high, and shows that golden eagle is quite common in Norway today. The indicator value is similar in the south as in the north of the country. Development needs include better population estimates through intensified monitoring, and a revision of the X~100~ values based on updated knowledge.     
```

## 2. About the underlying data

<!--# Give a general introduction the datasets you have used, and provide more specific details in the sub-chapters.  -->
The data comes from the Norwegian Nature index [@nater_naturindeks_2025], which did an evaluation of the golden eagle population based on expert opinion, leaning on the national carnivore monitoring program (www.rovdata.no). 

### 2.1 Spatial and temporal resolution and extent

<!--# Describe the temporal and spatial resolution and extent of the data used -->
Indicator values are available for the different assessment years in  _Naturindeksen_: 1990, 2000, 2010, 2014, 2019 and 2024. 

The indicator is calculated for southern and northern Norway. Here we report the values for five geographic regions by simply reusing the estimates from southern Norway and applying them to the other overlapping reagins ("Central", "East", "South", "West"). We also aggregate to a national estimate.

### 2.2 Sampling design

<!-- How was the data sampled? Describe potential sampling biases -->

> The data is based on expert opinion. No sampilng was infolved.

Reviewer: ```suggestion
The data is based on expert opinion. No sampilng was involved.
```

<!-- X~60~ is not specified and defaults to 60% of the X~100~ value.  -->

#### 4.2.1 Spatial resolution and validity

<!--# 

Describe the spatial resolution of the reference levels. E.g. is it defined as a fixed value for all areas, or does it vary. Also, at what spatial scale are the reference levels valid? For example, if the reference levels have a regional resolution (varies between regions), it might mean that it is only valid and correct to use for normalising local variable values that are first aggregated to regional scale. However, sometimes the reference levels are insensitive to this and can be used to scale variables at the local (e.g. plot) scale. 

 -->
 The reference values are the same everywhere.

## 5. Uncertainties
### 5.1 General uncertainties
<!--# Describe the main reasons why you might not trust 100% the final value to represent the given ecosystem characteristic for the give ecosystem type. What aspects of this condition metric are most sensitive to subjective decitions? Is the spatial sampling sufficient? Does the choice of modeling framework affect the values? Is the metric relevant do describe ecosystem condition for the give ecosystem type?  -->

> There is considerable uncertainties associated with how the X~100~value, as well as the current population estimates, as estimated based on expert opinion. There is no structured documentation for how these values have been estimated.

Reviewer: ```suggestion
There is considerable uncertainties associated with the X~100~value, as well as the current population estimates, as estimated based on expert opinion. There is no structured documentation for how these values have been estimated.
```

The reported uncertainties reflect the uncertainties around the true parameter means, i.e. they are inferrential uncertainties for the true value of the indicator for the given area and time combination.  

## 6. References

<!--# You can add references manually or use a citation manager and add intext citations as with crossreferencing and hyperlinks. See https://quarto.org/docs/authoring/footnotes-and-citations.html -->

::: {#refs}
:::

## 7. Import and prepare datasets

<!--# All the data import and preparation should be done in this chapter, before the analysis chapter. As a minimum, add code for importing data, and describe or show the data structure. You may also explain certain variables, what they represent, and their distributions -->

### 7.1 Naturindeks

<!--# Import and prepare the first dataset for analysis. Change the header from Dataset A to the name of the actual dataset. -->

We download the dataset from the Naturindeks database using the API. 

> The function giver read-only access without requiring a username and password.

Reviewer: ```suggestion
The function gives read-only access without requiring a username and password.
```
